operations-stdlib/src/commonMain/kotlin/string/Trim.kt

```diff
@@ -0,0 +1,54 @@
+package string
+
+import operation.StandardLogicOperation
+import utils.asList
+
+object Trim : StandardLogicOperation, StringUnwrapStrategy {
+    private const val ARGUMENTS_ARG_INDEX = 0
+    private const val TEXT_ARG_INDEX = 0
+    private const val CHAR_ARG_INDEX = 1
+    private const val MODE_ARG_INDEX = 2
+
+    override fun evaluateLogic(expression: Any?, data: Any?): Any? =
+        expression.asList.toOperationArguments()?.invokeTrim()
+
+    private fun TrimArguments.invokeTrim() = (this as? TrimArguments)?.modeBasedTrim(mode, char)
+
+    private fun List<Any?>.toOperationArguments(): TrimArguments? = runCatching {
+        val arguments = get(ARGUMENTS_ARG_INDEX).asList
```

Comment:
the input in your unit tests is nested too deeply and thats why you need second unwrapping. I think its unnecessary and unit tests should look like:
`operatorName to listOf("_cat_", "_", "start")`

Reply:
Done